src/lib/services/submit.services.ts

```diff
@@ -225,7 +225,23 @@ export const submitAddNodeProviderProposal = async ({
 		ProposalParams,
 		'neuronId'
 	>): Promise<SubmitProposalResult> => {
-		const { title, url, nodeProviderId, summary } = metadata;
+		const { url, nodeProviderId } = metadata;
+
+		const title = 'Add Node Provider: ' + metadata?.nodeProviderName;
+		const summary =
```

Comment:
Likewise about validation.

Nth: template literal for readability reason maybe?

Reply:
```		
const title = `Add Node Provider: ${metadata?.nodeProviderName}`;
const summary = `
  Register a node provider ${metadata?.nodeProviderName} in line with the announcement and discussion at ${metadata?.url}.
			
  The self-declaration documentation is available at ${metadata?.urlSelfDeclaration} 
  with SHA256 hash: ${metadata?.hashSelfDeclaration}.
			
  The proof of identity is available at ${metadata?.urlIdentityProof} 
  with SHA256 hash: ${metadata?.hashIdentityProof}.`;
```

I tried the above before but then in the NNS dapp it gets rendered like so:
<img width="529" alt="image" src="https://github.com/peterpeterparker/proposals.network/assets/79041287/cae124f2-6aa3-4660-8c0f-405f760458fb">

Ideally, viewers of the proposal can click the links directly. So this is why I kept it using string concatenation. Unless there is another way? wdyt?

Reply:
I think you can solve this by removing the space at the begins of the new lines

```
     const summary = `
NO_SPACE_HERE_Register a node...

The self-declaration docume...
with SHA256 hash...`;
```

Reply:
Ideally we probably also want to not have the template hardcoded but somewhere else, like in a Markdown file which is imported and which gets placeholder replace. Therefore the literal template being more handy. Not for this PR though.